src/snovault/schema_views.py

```diff
@@ -81,3 +82,16 @@ def schemas_map(context, request):
             profiles_map[type_info.name] = type_info.schema['id']
     profiles_map['@type'] = ['JSONSchemas']
     return profiles_map
+
+
+@view_config(route_name='schemas_titles', request_method='GET',
+             decorator=etag_app_version_effective_principals)
+def schemas_titles(context, request): # pylint: disable=unused-argument
```

Comment:
Inline comments technically get two or more spaces before the hash.
https://www.python.org/dev/peps/pep-0008/#inline-comments

Not a big deal, probably wouldn't have mentioned it if there wasn't another update.

Reply:
Thanks for the citation — I looked specifically for PEP8’s opinion on that and somehow never found that.

Reply:
yea, I wouldn't have known except some version of text editor I had in the past flagged it.